perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -114,7 +114,12 @@ sub bin_services_for_address {
         }
 
         my $containers = $self->_containers($property);
-
+        if ($service->{ServiceID} && $service->{ServiceID} == 1) {
```

Comment:
I don't think all sites will have a service with `ServiceID` == 1 (e.g. try `/waste/58247/` locally, seems that because it's a communal property there's no residual service).

Not sure what the best approach is here, I guess you could change this to "if the round schedule contains a week then update the calendar link to use that week" and just run it for all services. Or just take the first service that matches the pattern and use the week from that, and skip the check if the calendar link is already set?